components/developer/jenkins-core-weekly/Makefile

```diff
@@ -0,0 +1,90 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2016-2017 Jim Klimov
+#
+include ../../../make-rules/shared-macros.mk
+
+# NOTE: Before bumping to the new core version, take a look at
+#   https://jenkins.io/changelog/ (weekly)
+#   https://jenkins.io/changelog-stable/ (LTS)
+# for "weather report" (user votes on good/bad state of the release).
+# At some point it may make sense to deliver packaged LTS releases,
+# and the initial recipe supports that notion already just in case,
+# but no LTS issued so far includes updated libzfs.jar (v0.8+ needed).
+# Note that for LTS it would make sense to define indexed "additional
+# source" variables below, see e.g. docbook for inspiration.
+
+COMPONENT_NAME=		jenkins
+COMPONENT_MAJOR_VERSION=	2
+COMPONENT_MINOR_VERSION=	71
+COMPONENT_VERSION=	$(COMPONENT_MAJOR_VERSION).$(COMPONENT_MINOR_VERSION)
+COMPONENT_ARCHIVE=	$(COMPONENT_NAME)-$(COMPONENT_VERSION).war
+COMPONENT_FMRI=		developer/jenkins-core-weekly
+COMPONENT_CLASSIFICATION=	Development/Java
+COMPONENT_SRC=		$(COMPONENT_NAME)-$(COMPONENT_VERSION)
+COMPONENT_PROJECT_URL=	https://jenkins.io/changelog/
+COMPONENT_ARCHIVE_URL=\
+	http://mirrors.jenkins-ci.org/war/$(COMPONENT_VERSION)/jenkins.war
+# See $(COMPONENT_ARCHIVE_URL).sha256 e.g. for current weekly release, run
+#   wget -O - http://mirrors.jenkins-ci.org/war/latest/jenkins.war.sha256
```

Comment:
No idea, I don't use that - have to copy-paste instead :(

Also, wasn't the idea behind caching the SHAs in our Makefiles to protect against upstream re-publish of source+checksum while we are not aware of the change?